chat-plugins/mafia.js

```diff
@@ -297,7 +297,8 @@ class MafiaTracker extends Rooms.RoomGame {
 		this.allowRenames = false;
 		this.started = false;
 		this.ended = false;
-		this.theme = '';
+		/** @type {Object?} */
```

Comment:
Id rather not just say "Its any object or null", can you specify that its a mafia theme object or null?